pkg/windows/windows.go

```diff
@@ -783,6 +793,22 @@ func (vm *windows) newFileInfo(path string) (*payload.FileInfo, error) {
 	return &payload.FileInfo{Path: path, SHA256: sha}, nil
 }
 
+// removeHNSNetworks remove the HNS networks created by the hybrid-overlay configuration process
+func (vm *windows) removeHNSNetworks() error {
+	vm.log.Info("removing HNS networks")
+	cmd := "\"$net0 = Get-HnsNetwork | where { $_.Name -eq '" + BaseOVNKubeOverlayNetwork + "'};" +
```

Comment:
Please add a comment here mentioning the VIP created by WMCO also gets deleted when deleting the HNS networks.